e2e/testcases/cli_test.go

```diff
@@ -39,6 +46,11 @@ import (
 	"kpt.dev/configsync/pkg/testing/fake"
 )
 
+const (
+	KubeSystem          = "kube-system"
+	ResourceGroupSystem = "resource-group-system"
```

Comment:
Use RGControllerNamespace from pkg/api/configmanagement, as commented in https://github.com/GoogleContainerTools/kpt-config-sync/pull/134#discussion_r981813531.

Reply:
KubeSystem is also declared as `metav1.NamespaceSystem`.

Reply:
Done, const replaced by specific declarations.